coalib/bearlib/aspects/collections.py

```diff
@@ -7,7 +7,7 @@ class AspectList(list):
     List-derived container to hold aspects.
     """
 
-    def __init__(self, seq=()):
+    def __init__(self, seq=(), excludes=()):
```

Comment:
Change argument defaults to `None`. Otherwise you always do checks like `if excludes ...`. But there can be sequence that are `False` but can anyway contain elements. Best is always `seq=None, excludes=None` and then check `if excludes is None ...`. It's also written in some python docs...